microsoft/botframework-solutions
# [DevOps][TypeScript] Create yaml file with Pipeline configuration to setup CI

## Description

- Add Readme and images
### YAML Files added:
- Add Botbuilder-Skills.
- Add Botbuilder-Solutions. 
- Add Sample-Assistant. 
- Add Sample-Skil.l 
- Add Botskills. 

Related Issue #932 

## Testing Steps

1. Review the readme in `yaml` folder
2. Use the documentation to create a pipeline
3. Check that the new pipeline is working

## Checklist
